### PR TITLE
Feature/docker setup

### PR DESCRIPTION
**Pull request by medson10**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+version: '3.1'
```
Comment (grsabreu) [inline docker-compose.yml]: https://github.com/ContaQuanto/pinkman/blob/master/docker-compose.yml

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      DATABASE_PASSWORD: "summhub"
+    build: .
+    expose:
+      - "3000"
+    ports:
+      - "3000:3000"
+    depends_on:
+      - db
+    links:
+      - sqitch
```
Comment (fakenickels) [inline docker-compose.yml]: 🤔

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  db:
+    image: postgres:latest
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: "summhub"
+      POSTGRES_USER: "summhub"
+      POSTGRES_DB: "summhub_prod"
+    ports:
+      - "5432:5432"
+    expose:
```
Comment (fakenickels) [inline docker-compose.yml]: do not expose in prod

File: package.json [hunk 4]
```diff
@@ -54,7 +54,9 @@
   "scripts": {
     "start": "esy b && env $(cat .env | xargs) esy x Server.exe",
     "pesy": "bash -c 'env PESY_MODE=update pesy'",
-    "test": "esy x TestSummarize.exe"
+    "test": "esy x TestSummarize.exe",
+    "docker:prod": "docker-compose up",
+    "docker:dev": "docker-compose -f docker-compose.dev.yml up"
```
Comment (fakenickels) [inline package.json]: can you add the scripts to easily run the migration executing the knex thing?

File: docker-compose.yml (hunk 2 above)
Comment (medson10) [reply docker-compose.yml]: forgot to remove

File: package.json (hunk 4 above)
Comment (medson10) [reply package.json]: sure

File: docker-compose.dev.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+version: '3.1'
+
+services:
+  server:
+    env_file:
+      - ./.env
+    environment:
+      DATABASE_URL: "postgresql://0.0.0.0:5432/summhub_dev"
```
Comment (fakenickels) [inline docker-compose.dev.yml]: why not "postgresql://postgres:5432/summhub_dev"?

File: docker-compose.dev.yml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      POSTGRES_PASSWORD: "summhub"
+      POSTGRES_USER: "summhub"
+      POSTGRES_DB: "summhub_dev"
+    ports:
+      - "5432:5432"
+    expose:
+      - "5432"
+
+  sqitch:
+    image: sqitch/sqitch:latest
```
Comment (fakenickels) [inline docker-compose.dev.yml]: 🤔

**Review by georgelima — APPROVED** [review]

File: migrations/20190306154545_initial_schema.js [hunk 7]
```diff
@@ -0,0 +1,78 @@
+exports.up = function(knex, Promise) {
+  return Promise.all([
+    knex.schema.createTable('app_users', table => {
```
Comment (guilhermedecampo) [inline migrations/20190306154545_initial_schema.js:3]: top

File: docker-compose.dev.yml (hunk 5 above)
Comment (medson10) [reply docker-compose.dev.yml]: it is how they do in the docs 🤔

**Review by fakenickels — APPROVED** [review]

**Pull request merged**